2023/cmd/day07/main.go

```diff
@@ -13,28 +13,32 @@ import (
 
 var digitsRegexp = regexp.MustCompile("[[:digit:]]+")
 
-func getInput() (hands []*Hand) {
+func getInput() (game Game) {
```

Comment:
```suggestion
func getInput() (game *Game) {
```
Use a pointer instead to keep from copying the entire object, `game`...